src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java

```diff
@@ -48,4 +56,12 @@ public String create(MetadataCreateRequest request) throws HygieiaException {
         return entity.getId().toString();
     }
 
+    @Override
+    public DataResponse<Iterable<Metadata>> search(String searchKey, String value) throws HygieiaException {
+         LOGGER.info("MetdataService.search() :  Enter");
```

Comment:
This log line does not help debugging, as it has no information to the specific instance of search